lib/options.js

```diff
@@ -9,6 +9,24 @@ class Options {
 		this.epsilon = 0.01;
 		this.cycles = 100;
 
+		//Usage: discount_changes = times when to change, discounts = what discount to use
+
+		this.discount_changes = [];
```

Comment:
i've completely refactored how options/config/dem works, so if you refactor this we'll probably just have to refactor it again. maybe just tidy it up and simplify for now.